core/deployment/src/main/resources/dev-ui/qwc-qosdk-controllers.js

```diff
@@ -0,0 +1,65 @@
+import { LitElement, html, css, nothing} from 'lit';
+import { JsonRpc } from 'jsonrpc';
+import '@vaadin/grid';
+import '@vaadin/details';
+import '@vaadin/list-box';
+import '@vaadin/item';
+import {columnBodyRenderer} from '@vaadin/grid/lit.js';
+export class ControllersWebComponent extends LitElement {
+
+    jsonRpc = new JsonRpc(this);
+
+    constructor() {
+        super();
+    }
+
+    render() {
+        return html`
+            <vaadin-grid .dataProvider="${this._controllersProvider}">
+                <vaadin-grid-column path="name"></vaadin-grid-column>
```

Comment:
You can probably use sortable columns, that means the user can sort cols by name/class/namespace etc.